data/handwritten_text_recognition/create_lmdb_dataset.py

```diff
@@ -0,0 +1,87 @@
+""" a modified version of CRNN torch repository https://github.com/bgshih/crnn/blob/master/tool/create_dataset.py """
```

Comment:
Intel header need to be added here (in addition to the note about file origin).
```suggestion
""" 
 Copyright (c) 2021 Intel Corporation
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
      http://www.apache.org/licenses/LICENSE-2.0
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
 
a modified version of CRNN torch repository https://github.com/bgshih/crnn/blob/master/tool/create_dataset.py 
"""
```